NeuroID/src/advancedDeviceLib/java/com/neuroid/tracker/extensions/AdvancedDeviceExtension.kt

```diff
@@ -47,6 +40,7 @@ fun NeuroIDPublic.startSession(
     sessionID: String? = null,
     advancedDeviceSignals: Boolean
 ): SessionStartResult {
+    NeuroID.getInternalInstance()?.isAdvancedDevice = advancedDeviceSignals
```

Comment:
This function should NOT change the global variable, it should only override the global variable

Reply:
Not sure about this one. I think we should update NeuroID.isRN global variable (member) and we should not call checkThenCaptureAdvancedDevice() here in the extension function but only in the NeuroID.startSession() and NeuroID.start(). It is possible that we will request the ADV RID twice if we set the builder isRN to true an call startSession(sessionID, true).

Reply:
not overriding for now. will convene to simplify the flow later.